policy/modules/contrib/frr.te

```diff
@@ -0,0 +1,108 @@
+policy_module(frr, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type frr_t;
+type frr_exec_t;
+init_daemon_domain(frr_t, frr_exec_t)
+
+type frr_log_t;
+logging_log_file(frr_log_t)
+
+type frr_tmp_t;
+files_tmp_file(frr_tmp_t)
+
+type frr_lock_t;
+files_lock_file(frr_lock_t)
+
+type frr_conf_t;
+files_config_file(frr_conf_t)
+
+type frr_unit_file_t;
+systemd_unit_file(frr_unit_file_t)
+
+type frr_var_run_t;
+files_pid_file(frr_var_run_t)
+
+########################################
+#
+# frr local policy
+#
+allow frr_t self:capability { dac_override dac_read_search chown kill net_bind_service net_raw setgid setuid };
```

Comment:
I cannot find justification for dac_override/dac_read_search in this PR. Move chown in front of dac_.

Reply:
About dac_override I wrote in Ondra review, and about dac_read_search the vtysh need boot startup configuration with specific daemon, so I suppose that vtysh need read frr.conf .

Here I attach a example of full audit message about dac_read_search :
`type=PROCTITLE msg=audit(01/29/2021 10:00:36.033:3841) : proctitle=/usr/bin/vtysh -b -d zebra·
type=PATH msg=audit(01/29/2021 10:00:36.033:3841) : item=0 name=/etc/frr/frr.conf inode=262500 dev=fc:01 mode=file,640 ouid=frr ogid=frr rdev=00:00 obj=unconfined_u:object_r:frr_conf_t:s0 nametype=NORMAL cap_fp=none cap_fi=none cap_fe=0 cap_fver=0 cap_frootid=0·
type=CWD msg=audit(01/29/2021 10:00:36.033:3841) : cwd=/·
type=SYSCALL msg=audit(01/29/2021 10:00:36.033:3841) : arch=x86_64 syscall=openat success=yes exit=9 a0=0xffffff9c a1=0x5587f0b20780 a2=O_RDONLY a3=0x0 items=1 ppid=81191 pid=81243 auid=unset uid=root gid=root euid=root suid=root fsuid=root egid=root sgid=root fsgid=root tty=(none) ses=unset comm=vtysh exe=/usr/bin/vtysh subj=system_u:system_r:frr_t:s0 key=(null)·
type=AVC msg=audit(01/29/2021 10:00:36.033:3841) : avc:  denied  { dac_read_search } for  pid=81243 comm=vtysh capability=dac_read_search  scontext=system_u:system_r:frr_t:s0 tcontext=system_u:system_r:frr_t:s0 tclass=capability permissive=0·
type=AVC msg=audit(01/29/2021 10:00:36.033:3841) : avc:  denied  { dac_read_search } for  pid=81243 comm=vtysh capability=dac_read_search  scontext=system_u:system_r:frr_t:s0 tcontext=system_u:system_r:frr_t:s0 tclass=capability permissive=0`